nncf/experimental/post_training/statistics/statistic_point.py

```diff
@@ -75,10 +75,10 @@ def iter_through_statistic_points_in_target_node(self, target_node_name: str,
             if statistic_point_condition_func(_statistic_point):
                 yield _statistic_point
 
-    def iter_through_algorithm_tensor_collectors_in_target_node(self, target_node_name: str,
-                                                                statistic_point_condition_func: Callable[
-                                                                    [StatisticPoint], bool],
-                                                                algorithm: 'PostTrainingAlgorithms'):
+    def get_algo_statistics_for_node(self, target_node_name: str,
```

Comment:
Return type hint missing. Consider removing quotes from `'PostTrainingAlgorithms'` type hint here and everywhere. If you get circular imports due to this, refactor your code files so that they are more granular.

Reply:
Return type hint added. 
Ticket regarded the circular imports - 94101